docs/reference/using/cli/gel_instance/gel_instance_backup.rst

```diff
@@ -0,0 +1,65 @@
+.. _ref_cli_gel_instance_backup:
+
+
+===================
+gel instance backup
+===================
+
+Backup a |Gel| instance to disk.
+
+.. cli:synopsis::
+
+    gel instance backup [<options>]
+
+
+Description
+===========
+
+The :gelcmd:`instance backup` command creates a backup of the specified
+insatnce. This backup can be restored using the
+:ref:`ref_cli_gel_instance_restore` command.
+
+The tool will automatically determine whether to perform a full or an
+incremental backup. The goal is to balance the backup size with the restore
+speed. So the rule of thumb is that an incremental backup will be created if
```

Comment:
Is it "restore speed" we're optimizing or "backup speed"?

Reply:
I think we should say backup speed. We prefer to take snapshots as quickly as we can.